src/rust/engine/fs/brfs/src/main.rs

```diff
@@ -767,23 +765,19 @@ async fn main() {
 
     let local_only_store =
         Store::local_only(runtime.clone(), store_path).expect("Error making local store.");
-    let store = match args.value_of("server-address") {
+    let store = match args.get_one::<String>("server-address") {
         Some(address) => local_only_store
             .into_with_remote(RemoteStoreOptions {
                 provider: RemoteProvider::Reapi,
                 store_address: address.to_owned(),
-                instance_name: args.value_of("remote-instance-name").map(str::to_owned),
+                instance_name: args.get_one::<String>("remote-instance-name").cloned(),
                 tls_config,
                 headers,
                 chunk_size_bytes: 4 * 1024 * 1024,
                 timeout: std::time::Duration::from_secs(5 * 60),
                 retries: 1,
-                concurrency_limit: args
-                    .value_of_t::<usize>("rpc-concurrency-limit")
-                    .expect("Bad rpc-concurrency-limit flag"),
```

Comment:
These `.expect`s are no longer needed - `get_one()` will panic appropriately already, with more information than this message provides...